endaq/device/schemata/command-response.xml

```diff
@@ -49,6 +50,7 @@
         <UIntegerElement name="CommandIdx" id="0x88" multiple="0" mandatory="1">Incrementing index, will bump by one for each update to this data structure, starting at 0 after device reset</UIntegerElement>
         <!-- Legacy commands, duplicated within EBMLCommand for serial use (which requires the outer EBMLCommand element) -->
         <MasterElement name="RecStart" id="0x7273" mandatory="0" multiple="0">"rs" Initiate a recording as if button was pressed</MasterElement>
+        <MasterElement name="RecStop" id="0x7275" mandatory="0" multiple="0">"ro" Stop a recording as if button was pressed</MasterElement>
```

Comment:
`0x7275` is ASCII `ru`; minor, but you should probably update the description, or change the ID to `0x726F`.